Refactoring aggregation analyses

anmolsingh20 opened the pull request:
Hi Ralf, could you please have a look. I will probably need to rebase again, but it is a moving target, so will do after a review. There is an additional pull request on the resources repo to complete this change.

Commit:
Rename directory 'R' to 'analyses' to support general analyses independent
of chosen language.

Signed-off-by: Anmol Singh <anmol.singh@bmw.de>

Commit:
Use a default location for patch responses and extend
pasta prepare_evaluation to handle data processing

Use default location for patch responses and extend review analysis with
placeholders for subsequent processing. It is essential to break the
processing into several steps to avoid memory issues. Additionally,
these parts of the analysis would use pasta classes and moving them
outside the package structure (to analyses, e.g.) would lead
to unclean package imports.


Signed-off-by: Anmol Singh <anmol.singh@bmw.de>

rralf commented:
Hi Anmol,
I rebased to next, and accepted both patches. For the second one, I referenced the new commit hash of the resources submodule, and removed all changes that are not related to the change of the location of the configuration. The reason: I don't understand why you import pandas, flat_table or dask. There are currently no users of those libraries.

anmolsingh20 commented:
> Hi Anmol,
> I rebased to next, and accepted both patches. For the second one, I referenced the new commit hash of the resources submodule, and removed all changes that are not related to the change of the location of the configuration. The reason: I don't understand why you import pandas, flat_table or dask. There are currently no users of those libraries.

I missed removing those imports while rolling back some other changes, so it is good that you removed them. They will follow in the next patch. Thanks!
